web-client/src/presenter/actions/CaseDetail/completeMessageAction.ts

```diff
@@ -21,7 +21,8 @@ export const completeMessageAction = async ({
     .completeMessageInteractor(applicationContext, {
       messages: [
         {
-          messageBody: form?.message,
+          messageBody: ((form as { message?: string })?.message ||
+            '') as string,
```

Comment:
The `as string` assertion shouldn't be necessary here.

You could use annotation instead:

```typescript
const form: { message?: string } = get(state.modal.form);
```
and then just
```typescript
messageBody: form?.message || '',
```